src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/06-sbt-new-and-Templates.md

```diff
@@ -0,0 +1,133 @@
+---
+out: sbt-new-and-Templates.html
+---
+
+  [giter8]: http://www.foundweekends.org/giter8/
+  [foundweekends]: http://www.foundweekends.org/
+  [CC0]: https://creativecommons.org/publicdomain/zero/1.0/
+
+sbt new and Templates
+---------------------
+
+sbt 0.13.13 adds `new` command, which helps create new build definitions.
+The `new` command is extensible via a mechanism called the template resolver.
+
+### Trying new command
+
+To get started, you need to first install sbt 0.13.13 launcher or above.
```

Comment:
I think that should be covered somewhere else, like getting started guide's setup section.